srcpkgs/mediainfo/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'mediainfo'
 pkgname=mediainfo
-version=18.08
-revision=2
-wrksrc="MediaInfo"
+version=18.08.1
+revision=1
+wrksrc=MediaInfo
+configure_args="--with-wx-config=/usr/bin/wx-config-gtk3"
```

Comment:
Should probably be just `--with-wx-config=wx-config-gtk3` so that it uses the wrapper in cross builds.

Reply:
I think it might be necessary to also add `wxWidgets-gkt3-devel` to hostmakedepends as wx-config returns absolute paths without respecting `XBPS_CROSS_BASE`.

Reply:
Sorry if I misunderstand, but the wrapper does add `$XBPS_CROSS_BASE`:
```
exec ${XBPS_CROSS_BASE}/usr/bin/wx-config-gtk3 --prefix=${XBPS_CROSS_BASE}/usr "$@"
```

Reply:
I did initially had an issue with just `wx-config-gtk3` but it was triggered by smth else and I just forgot to revert iy. Should be fine now

Reply:
@newbluemoon I had just a little time to test it earlier and saw that the headers were not found. If the issue was something else and the wrapper works, that's fine of course.